lib/constants.go

```diff
@@ -1353,7 +1359,7 @@ var DeSoMainnetParams = DeSoParams{
 
 	// The congestion factor determines when we will start to increase or decrease fees.
 	// We set the congestion factor to 90% for past blocks and mempool. This makes it so that we will
-	// start to increase fees when the past N blocks (DefaultMempoolFeeEstimatorNumPastBlocks) are
+	// start to increase fees when the past N blocks (DefaultMempoolPastBlocksCongestionFactorBasisPoints) are
```

Comment:
```suggestion
	// start to increase fees when the past N blocks (DefaultMempoolFeeEstimatorNumPastBlocks) are
```

Reply:
Resolved in https://github.com/deso-protocol/core/pull/1261. Thanks for fixing this